mappings/net/minecraft/client/util/math/MatrixStack.mapping

```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/unmapped/C_cnszsxvd net/minecraft/client/util/math/MatrixSta
 		ARG 2 y
```

Comment:
I think this whole file should be updated to match fabric's mappings. Calling the `modelMatrix` `positionMatrix` just makes more sense for this file, because `MatrixStack` doesn't only work with model coordinates. This means that `multiplyMatrix` should be changed to `multiplyPositionMatrix`, `getModel` should be replaced with `getPositionMatrix`, etc.

Reply:
Yes, but model is how it's used in minecraft and it's not used for the projection or view matrices. I'll think about it.